.github/workflows/ci-pr-checks.yaml

```diff
@@ -49,7 +49,6 @@ jobs:
       - name: Run lint checks
         uses: golangci/golangci-lint-action@v9
         with:
-          version: 'v2.1.6'
```

Comment:
what is the rationale for removing the specific version? will it run with some `latest v2`?